userguide/tasks.rst

```diff
@@ -879,7 +879,8 @@ and click |ui-add|. This wizard simplifies creating a replication to
 This first screen provides options to configure the connection between
 |rpln-sys1| and |rpln-sys2|.
 
-Enter a descriptive :guilabel:`Name` for this replication.
+Enter a descriptive :guilabel:`Name` for this replication. Replication
+names, SSH Connection names, and SSH Keypair names must be unique.
```

Comment:
Why are we mentioning here that SSH connections and keypair names also have to be unique?  Does that relate to this section, or should it be mentioned elsewhere when those names are discussed?

Reply:
Good point. I forgot to update those specific sections. I'll update them now.

However, I still think it's relavent here as well because SSH connections or SSH key pairs may be needed to set up a replication task.